cmake/submodules.cmake

```diff
@@ -2,7 +2,11 @@
 IF(GIT_EXECUTABLE AND EXISTS "${CMAKE_SOURCE_DIR}/.git")
   EXECUTE_PROCESS(COMMAND "${GIT_EXECUTABLE}" config --get cmake.update-submodules
                   WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-                  OUTPUT_VARIABLE cmake_update_submodules)
+                  OUTPUT_VARIABLE cmake_update_submodules
+                  RESULT_VARIABLE git_config_get_result)
+  IF(git_config_get_result EQUAL 128)
+    SET(cmake_update_submodules no)
+  ENDIF()
   IF(cmake_update_submodules MATCHES no)
```

Comment:
This following form looks more natural to me:
```
IF(git_config_get_result EQUAL 128 OR cmake_update_submodules MATCHES no)
```
But this one is probably fine as well.

Reply:
You're right. I like that. Updated.